projects/kubernetes-sigs/vsphere-csi-driver/Makefile

```diff
@@ -1,12 +1,14 @@
 BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
 GIT_TAG:=$(shell cat GIT_TAG)
-GOLANG_VERSION?="1.15"
+GOLANG_VERSION?="1.17"
 
 REPO=vsphere-csi-driver
 REPO_OWNER=kubernetes-sigs
 
 BINARY_TARGET_FILES=vsphere-csi-driver vsphere-csi-syncer
 SOURCE_PATTERNS=./cmd/vsphere-csi ./cmd/syncer
+BINARY_TARGET_FILES_BUILD_ALONE=vsphere-csi-syncer
```

Comment:
@jaxesn We weren't setting the [version LDFLAGS](https://github.com/kubernetes-sigs/vsphere-csi-driver/blob/v2.6.0/Makefile#L91-L92) for these Go builds, and they differ for the driver and syncer binaries, so building this separately.